aim/agent/aid/universes/aci/tenant.py

```diff
@@ -436,42 +594,29 @@ def _push_aim_resources(self):
                                 aim_object.pre_existing = True
                             to_push = self.to_aci_converter.convert(
                                 [aim_object])
```

Comment:
I see this debug log "LOG.debug('%s AIM object %s in APIC' % (method, repr(aim_object)))" was removed. Can we put it back then as I don't see that info is being kept somewhere else so.

Reply:
Will put back.